### ISSUE TITLE
[DOC] Include Golang RPC client 

### ISSUE THREAD

**Issue by mllnd** [near/docs#734]
We've developed a Golang RPC library to interact with accounts and smart contracts on NEAR blockchain network. Perhaps it would be beneficial for future developers to have it listed on the **Getting Started** page as well? We'd be more than happy to contribute official documentation as well, if necessary. We're also actively maintaining it, since it is a central piece of our backend API.

https://github.com/eteu-technologies/near-api-go

**Comment by thisisjoshford** [comment]
Awesome! Love this @mllnd :) Would you be able to add more documentation on how to use this SDK? Similar to [near-api-js quick reference](https://docs.near.org/docs/api/naj-quick-reference)?

**Comment by mllnd** [comment]
Yup, we can provide all relevant documentation for the Golang package. How should we proceed with this?

**Comment by thisisjoshford** [comment]
> Yup, we can provide all relevant documentation for the Golang package. How should we proceed with this?

I would say enhance your repo's readme or create an examples subfolder with commands like `near-api-js`.
https://github.com/near/near-api-js/blob/master/examples/quick-reference.md
https://github.com/near/near-api-js/tree/master/examples/cookbook
